templates/projects/list.tmpl

```diff
@@ -1,5 +1,5 @@
 <div role="main" aria-label="{{.Title}}" class="page-content repository projects">
-	<div class="ui container">
+	<div class="ui{{if .ContextUser.IsOrganization}} container{{end}}">
```

Comment:
That's too hacky. Why "IsOrganization"  means "container"?

Reply:
If possible, remove the container.

Reply:
Yes, it is hacky beause we are sharing this between repository and orgransation page.

Reply:
Ok, it might be an acceptable compromise then. Didn't check the details.

Reply:
My opinion is that these `if-else` tricks are too fragile, so I can't vote my approval, while I won't block if others like it and approve.

Reply:
Looking for altenates suggestions to take this Pr forward.